lib/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(LIBFACTER_COMMON_SOURCES
     "src/facts/external/yaml_resolver.cc"
     "src/facts/map_value.cc"
     "src/facts/resolver.cc"
+    "src/facts/resolvers/augeasversion_resolver.cc"
```

Comment:
Can we name this the "augeas resolver" that populates any augeas-related facts?  That way it's more generic in case in the future we want it to report on more than just the version.